files/default/kagent_utils/kagent_utils/monitoring/service.py

```diff
@@ -56,12 +56,23 @@ def set_state(self, state):
         finally:
             self.state_lock.release()
 
+    def parse_active_from_status(self, status_str):
+        for s in status_str.split("\n"):
+            s = s.strip()
+            if s.startswith("Active"):
+                l = s.split(" ")[1:]
+                return " ".join(l)
+        return ""
+    
     def alive(self, currently=False):
         try:
             self.LOG.debug("Checking status of %s", self.name)
             command = ['systemctl', 'is-active', '--quiet', self.name]
             self._exec_check_call(command, stderr=subprocess.STDOUT)
-            msg = "Service {0} is alive".format(self.name)
+            command = "systemctl status {0}".format(self.name)
```

Comment:
do we have a better way to handle this instead of using status and parsing the string?

Reply:
actually, since you are printing the whole line - I guess it is fine so no problem with this one

Reply:
Initially I was getting the start time using `systemctl show --property=ExecMainStartTimestamp --value UNIT_NAME` but then in Python2 it is quite difficult to parse the returned time in order to compute a prettier "XX minutes/days ago".
So my final iteration was parsing the output from `systemctl status` which already gives a pretty output.

Reply:
Yeah, in the end the output is `2023-08-08 08:29:18 INFO [service/alive] Service nodemanager is active (running) since Thu 2023-08-03 15:17:42 UTC; 4 days ago`